draft-ietf-quic-transport.md

```diff
@@ -1790,8 +1790,13 @@ abandons its attempt to validate the path.
 
 Endpoints SHOULD abandon path validation based on a timer. When setting this
 timer, implementations are cautioned that the new path could have a longer
-round-trip time than the original.  A value of three times the current
-Retransmittion Timeout (RTO) as defined in {{QUIC-RECOVERY}} is RECOMMENDED.
+round-trip time than the original.  A value of three times the larger of the
+current Probe Timeout (PTO) or the initial timeout (that is, kInitialRtt*2) as
```

Comment:
```suggestion
 current Probe Timeout (PTO) or the initial timeout (that is, 2*kInitialRtt) as
```

Reply:
grr, leading spaces on suggestions are the worst